packages/ui/src/kit/VTooltip/VTooltip.theme-default.scss

```diff
@@ -0,0 +1,85 @@
+.v-tooltip.theme-default {
+  position: relative;
+}
+
+.v-tooltip-fade-enter-active,
+.v-tooltip-fade-leave-active {
+  transition: opacity .2s;
+}
+.v-tooltip-fade-enter,
+.v-tooltip-fade-leave-to {
+  opacity: 0;
+}
+
+
+.v-tooltip.theme-default .v-tooltip-container {
+  position: absolute;
+  font-size: 14px;
+  padding: 9px 16px 12px;
+  background-color: var(--endpass-ui-color-black);
+  color: var(--endpass-ui-color-white);
+  border-radius: 2px;
+  min-width: 172px;
+}
+
+.v-tooltip.theme-default .v-tooltip-triangle {
+  position: absolute;
+  width: 0;
+  height: 0;
+}
+
+.v-tooltip.position-top-left .v-tooltip-triangle {
+  top: -5px;
+  left: 12px;
+}
+.v-tooltip.position-top-left .v-tooltip-container {
+  left: 50%;
+  margin-left: -16px;
+  margin-top: 9px;
+}
+
+.v-tooltip.position-top-right .v-tooltip-triangle {
+  top: -5px;
+  right: 12px;
+}
+.v-tooltip.position-top-right .v-tooltip-container {
+  right: 50%;
+  margin-right: -16px;
```

Comment:
Are you sure that it is ok to combine `margin-right` with `right` in `abosolute` positioned element? Use `calc()` function or `transform`.

Reply:
i'm sure, it's oldschool story